src/parameterizations/lateral/MOM_thickness_diffuse.F90

```diff
@@ -505,7 +505,7 @@ subroutine thickness_diffuse(h, uhtr, vhtr, tv, dt, G, GV, US, MEKE, VarMix, CDp
         enddo
 
         do j=js,je ; do i=is,ie
-          MEKE%Kh_diff(i,j) = MEKE%Kh_diff(i,j) / MAX(1.0,G%bathyT(i,j))
```

Comment:
The change on this line does fix a dimensional inconsistency in this expression, and I agree that it is an improvement.  However, this change is going to have a conflict with PR #1484, which has already been merged into dev/gfdl, but not into main. I think that the latter change (in PR #1484) is the one we should ultimately keep.

  One note on style here...  I think that it might be preferable to use 1.0*GV%m_to_H to just GV%m_to_H, as the former emphasizes that this is a hard-coded 1.0 m length that is being converted, and it makes it easier to identify it later in searches for hard-coded constants.  Of course, any decent compiler would see these two constructs as equivalent.

  My recommendation here is to accept this code into main as is, and then sort out this conflict when we merge main back to dev/gfdl.